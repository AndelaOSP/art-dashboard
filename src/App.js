import React, { Component } from 'react';
import { Router, Route, Switch } from 'react-router-dom';
import { Provider } from 'react-redux';
import { createBrowserHistory } from 'history';
import './App.css';

import RoutesComponent from './_components/RoutesComponent';

import store from './_store';
const history = createBrowserHistory();


class App extends Component {
  render() {
    return (
      <div className="App">
      <Provider store={store}>
        <Router history={history}>
          <Switch>
            <Route path={`/`} component={RoutesComponent} />
          </Switch>
        </Router>
      </Provider>
    </div>
    );
  }
}

export default App;
