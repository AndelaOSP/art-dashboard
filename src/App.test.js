import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

import localStorageMock from './_mock/localStorage';

window.localStorage = localStorageMock;

it('renders without crashing', () => {
  expect(() => shallow(<App />)).not.toThrow();
});
