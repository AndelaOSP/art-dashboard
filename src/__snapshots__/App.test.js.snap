// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppComponent/> renders without crashing 1`] = `
<div>
  <div
    class="app landing-overlay background"
  >
    <div
      class="ui container"
    >
      <img
        src="http://res.cloudinary.com/damc3mj5u/image/upload/v1526571584/logo_uw39tc.png"
        alt="Andela logo"
        class="ui centered image"
      />
      <div
        class="ui inverted header landing-heading"
      >
        ART
      </div>
      <h1
        class="ui inverted header"
      >
        Andela Resource Tracker
      </h1>
      <button
        class="ui large button google-button"
        role="button"
      >
        <img
          src="http://res.cloudinary.com/damc3mj5u/image/upload/v1526571608/google-logo_jjjjqs.svg"
          alt="Google logo"
          class="ui left floated image"
        />
        Sign in with Google
      </button>
    </div>
  </div>
</div>
`;
